### PR TITLE
fruits and vegetables box

### PR DESCRIPTION
**Pull request by manusmd**

### CODE REVIEW

**Review by riittagerhards — COMMENTED** [review]
lgtm

File: src/components/button/button.css [hunk 1]
```diff
@@ -1,8 +1,8 @@
 .btn {
   border-radius: 20px;
```
Comment (leonardpaulus) [inline src/components/button/button.css:2]: use relative values

File: src/components/favorite-card/favorite-card.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  background-color: #eff7ee;
+  color: #1c3418;
+}
+
+.favorite-card--snacks {
+  background-color: #fff8eb;
+  color: #4d3200;
+}
+
+.selection {
```
Comment (lmachens) [inline src/components/favorite-card/favorite-card.css:27]: I am not sure about the naming. I would expect that an element with the class `selection` displays something, the user already selected.

File: src/components/favorite-card/favorite-card.html [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+      <path
+        d="M17.305 39.5801C17.305 40.4766 16.5782 41.2033 15.6818 41.2033C14.7853 41.2033 14.0586 40.4765 14.0586 39.5801C14.0586 38.6836 14.7853 37.057 15.6818 37.057C16.5782 37.057 17.305 38.6836 17.305 39.5801Z"
+        fill="#660C00"
+      />
+      <path
+        d="M15.9881 39.5801C15.9881 39.0451 16.2471 38.25 16.6465 37.6957C16.3768 37.3215 16.0429 37.0568 15.6816 37.0568C14.7851 37.0568 14.0583 38.6836 14.0583 39.5799C14.0583 40.4765 14.7852 41.2031 15.6816 41.2031C16.0435 41.2031 16.3766 41.0832 16.6465 40.883C16.2478 40.5875 15.9881 40.1148 15.9881 39.5801Z"
+        fill="#660C00"
+      />
+    </svg>
+    Fruits
```
Comment (JulianSchumacher99) [inline src/components/favorite-card/favorite-card.html:119]: Maybe you can add a span tag to the word :)

**Review by lmachens — CHANGES_REQUESTED** [review]
good job 🚀

but you removed a file `package-lock.json` which is important for `npm install`. 
the button changes should not be part of this pull request.

File: src/components/button/button.css [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
   font-size: 1.2rem;
-  color: white;
-  background: #016b6b;
+  color: black;
+  background: #91c788;
   border: none;
   padding: 10px 20px;
```
Comment (leonardpaulus) [inline src/components/button/button.css:7]: maybe use relative values

**Review by markobrkovic — COMMENTED** [review]
lgtm

**Review by JulianSchumacher99 — CHANGES_REQUESTED** [review]
Looks nice as always :)

**Review by lmachens — APPROVED** [review]
gj!

File: src/components/favorite-card/favorite-card.css (hunk 2 above)
Comment (manusmd) [reply src/components/favorite-card/favorite-card.css:27]: thx I changed it to "container"

File: src/components/favorite-card/favorite-card.html (hunk 3 above)
Comment (manusmd) [reply src/components/favorite-card/favorite-card.html:119]: thx I added span

File: src/components/button/button.css (hunk 1 above)
Comment (manusmd) [reply src/components/button/button.css]: deleted btn css from pull request

**Pull request merged**